README.md

```diff
@@ -2,27 +2,24 @@
 
 Repo containing the initLab's IVR telephony system, which enables opening of doors.
```

Comment:
Next time can we have a normal branch name - `20230904-fixes` is definitely not informative.

Reply:
OK